spk/gerrit/src/dsm-control.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# Package
+PACKAGE="gerrit"
+DNAME="Gerrit"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${GIT_DIR}/bin:${PATH}"
+USER="gerrit"
+PID_FILE="${INSTALL_DIR}/logs/gerrit.pid"
+LOG_FILE="${INSTALL_DIR}/logs/error_log"
+
+start_daemon ()
+{
+    su ${USER} -c "PATH=${PATH} ${INSTALL_DIR}/bin/gerrit.sh start"
```

Comment:
I may have misunderstood the purpose of the various manifests and makefiles... What's the best procedure for a package like this, where the actual SPK contains only a limited subset of what's required to run the application (but is automatically bootstrapped into the full application)?

The Gerrit install/upgrade process is not great from the packaging side, I admit... This 'init after install' approach is roughly what is used by the semi-official deb/rpm packages though: https://gerrit.googlesource.com/gerrit-installer/+/refs%2Fheads%2Fmaster